protos/configuration.proto

```diff
@@ -105,6 +105,11 @@ message RaftConfig {
 
   // Take a snapshot when cumulative data since last snapshot exceeds a certain size in bytes.
   uint64 snapshot_interval_size = 5;
+
+  // The maximal Raft ID in the cluster, during the entire lifetime of the cluster.
+  // This is computed internally by the server and is read only for clients.
+  // The Raft ID of a new peer added to the cluster mast be higher than this value.
```

Comment:
It is not a requirement of fabric, it is a requirement of Raft. 
```
Note: An ID represents a unique node in a cluster for all time. A given ID MUST be used only once even if the old node has been removed. This means that for example IP addresses make poor node IDs since they may be reused. Node IDs must be non-zero.
```
see https://pkg.go.dev/github.com/coreos/etcd/raft#section-readme

Fabric has a different approach for dealing with this requirement, by internally dealing and managing these IDs. 

We chose a simpler approach where the definition of the Raft ID is exposed to the admin. All the admin has to do when adding a node it to read the ClusterConfig, and add a NodeConfig and a PeerConfig.
The PeerConfig has a RaftID field, which he admin assigns. All he has to do is set a value > `ConsensusConfig.RaftConfig.MaxRaftID`.

He does not have to change the read `ConsensusConfig.RaftConfig.MaxRaftID` or set it back to zero.

Reply:
I will update the comment to reflect this semantic.